usersync/usersync.go

```diff
@@ -36,3 +36,20 @@ type CookieSyncBidders struct {
 	NoCookie     bool          `json:"no_cookie,omitempty"`
 	UsersyncInfo *UsersyncInfo `json:"usersync,omitempty"`
 }
+
+type syncer struct {
```

Comment:
This conflicts with the definition of `syncer` in `usersync/usersyncers/syncer.go` as well as being broken since this definition of the struct does an infinite recursion for `syncer.GDPRVendorID()`.